packages/vite/src/node/__tests__/plugins/importGlob/__snapshots__/fixture.spec.ts.snap

```diff
@@ -52,6 +52,7 @@ export const cleverCwd2 = /* #__PURE__ */ Object.assign({"./modules/a.ts": () =>
 
 
 });
+export const customBase = /* #__PURE__ */ Object.assign({"fixture-a/modules/a.ts": () => import("./modules/a.ts"),"fixture-a/modules/b.ts": () => import("./modules/b.ts"),"fixture-a/modules/index.ts": () => import("./modules/index.ts"),"fixture-a/sibling.ts": () => import("./sibling.ts")});
```

Comment:
I think the expected output is:
```ts
export const customBase = /* #__PURE__ */ Object.assign({"modules/a.ts": () => import("./modules/a.ts"),"modules/b.ts": () => import("./modules/b.ts"),"modules/index.ts": () => import("./modules/index.ts"),"sibling.ts": () => import("./sibling.ts")});
```
(the difference is `fixture-a/` in the key)

Normal relative path (not the root relative path, which starts with `/`) in `import.meta.glob` is resolved from the file that contains the `import.meta.glob` instead of the root.

Reply:
It's good. I fixed it.
[5db0640](https://github.com/vitejs/vite/pull/18510/commits/5db0640b20913245a861dad03d2e31529d127bf2)